Sources/NSDictionary+Safety.swift

```diff
@@ -4,11 +4,11 @@ extension NSDictionary {
     @nonobjc
     func safeValue(forKeyPath keyPath: String) -> Any? {
         var object: Any? = self
-        var keys = keyPath.split(separator: ".").map(String.init)
+        var keys = keyPath.split(separator: ".")
 
         while !keys.isEmpty, let currentObject = object {
             let key = keys.remove(at: 0)
-            object = (currentObject as? NSDictionary)?[key]
+            object = (currentObject as? NSDictionary)?[String(key)]
```

Comment:
Looks like this change isn't necessary it still works?

Reply:
Sorry for late response. I'm not sure as the root cause of the error is unclear. I will revisit on Xcode 12 beta 3